src/stores/districtInformations.ts

```diff
@@ -0,0 +1,94 @@
+import { ref } from 'vue'
+import type { Ref } from 'vue'
+import { defineStore } from 'pinia'
+import type { Viewpoint } from '@vcmap/core'
+import type { Feature } from 'ol'
+
+export const useDistrictStore = defineStore('districtInformations', () => {
+  const districtCode: Ref<number> = ref(0)
+  const districtName: Ref<string> = ref('')
+  const districtConsumption: Ref<number> = ref(0)
+  const districtProduction: Ref<number> = ref(0)
+  const districtNumberInstallations: Ref<number> = ref(0)
+  const districtFeature: Ref<Feature | null> = ref(null)
```

Comment:
districtFeature is not use in the project, maybe you can delete it 
If you delete it you can delete the function set too:  setFeatureDistrict